blocklist-client/README.md

```diff
@@ -14,12 +14,29 @@ You can run the blocklist client by providing the required environment variables
 #### Required Environment Variables
 - BLOCKLIST_CLIENT_RISK_ANALYSIS__API_URL=`<provider-url>`
 - BLOCKLIST_CLIENT_RISK_ANALYSIS__API_KEY=`<your_api_key>`
+- BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS=`<true|false>`
+
+Currently we implemented two risk providers, which can be chosen via the `BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS` environment variable. If set to `true`, the [sanctions](https://public.chainalysis.com/docs/index.html#introduction) provider will be used, otherwise the risk provider will be used. We currently lost information about the risk provider, so only with the `BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS` set to `true` the client will work.
+
```

Comment:
```suggestion
Currently this client implements two risk providers, which can be selected via the `BLOCKLIST_CLIENT_RISK_ANALYSIS__USE_SANCTIONS` environment variable. If set to `true` (the default), the [sanctions](https://public.chainalysis.com/docs/index.html#introduction) provider will be used, otherwise the risk provider will be used. 
```